spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -619,8 +619,10 @@ def main(argv: Sequence[str]):
     # Loop through all compressed levels (compute one MSCC per compressed level)
     rows = []
     for idx in compressed_levels_dict.keys():
+        # Get compressed level and slice
         level = list(compressed_levels_dict[idx].keys())[0]  # TODO change if more than one level
-        printv(f'\nCompression at level {int(level)}', verbose=verbose, type='info')
+        slice = list(compressed_levels_dict[idx].values())[0][0]
```

Comment:
Nitpick: We already access `list(compressed_levels_dict[idx].values())[0]` later on in this code block (`slice_avg`), so we're sort of duplicating that a little?

Reply:
Side note: This whole call is a little weird, what with the `[0][0]` and the access of `.keys()` and `.values()`. I think the weirdness is due to the structure of `compressed_levels_dict`:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/9144a030-4363-4f26-93c6-b93ab076e7df)

It's a dict of dicts of lists, which is a bit of a strange structure to use... I presume this is because A) The compression site could theoretically span multiple levels, and B) The compression site could theoretically span multiple slices within the same level?

But, right now we ignore all of that and simply take the first level + first slice...

I think this could maybe be refactored to be a bit simpler? But, tbh, it's outside the scope of this PR, so I don't think there's really too much to worry about.

Reply:
I've made some very nitpicky changes to simplify and deduplicate the access of the slice number.

Reply:
> A) Each compression site (0, 1, 2...) could theoretically span multiple levels (3.0/4.0, 4.0/5.0, etc.), and B) The compression site could theoretically span multiple slices within the same level (e.g. [159, 160])?

Correct for both. Currently, we simplify this by labeling only the maximum compressed slice for each compression (step 4 in [our tutorial](https://spinalcordtoolbox.com/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generate-necessary-inputs.html#label-spinal-cord-compression-sites)). In other words, even though compression spans several slices, we only put the label on one.